src/components/popover/Popover.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Popover } from './Popover'
+
+const meta = {
+  title: 'Components/Popover',
+  component: Popover,
+  parameters: {
+    layout: 'centered',
+  },
+  argTypes: {
+    placement: {
+      control: 'radio',
+      options: ['top', 'right', 'bottom', 'left'],
+    },
+    trigger: {
+      control: 'radio',
+      options: ['hover', 'click'],
+    },
+  },
+} satisfies Meta<typeof Popover>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    title: 'Did you know?',
+    content: (
```

Comment:
Have you found a nice way to be able to customize this content from a storybook control? I was thinking of looking into I think it's called like "mapped attributes" or something where you can specify a couple of different variations and then pick between them, without having to have a whole story dedicated to a few variants of one story

Reply:
Hmm I'm not following. At a high level, what would the different variations be that we would want to pick between?